client/FriendsList.js

```diff
@@ -42,6 +42,7 @@ FriendsList.prototype.handleUpdate = function(message) {
 	var friend = this.friends.find({id: message.id})
 	if(friend) friend.update(message)
 	else this.addFriend(new Friend(this.map, message))
+	if(message.stale) new Toast(message.name + ' has gone stale')
```

Comment:
These toasts are going to be pretty frequent if a user is going back and forth from this app and another. We should probably just change the icon and be done with it.